README.md

```diff
@@ -1,2 +1,47 @@
 # comp130-journal
 Base repository for COMP130 research journal
+
+[] are used to denote the sourced paper with the number corresponding to the paper in the source list.
```

Comment:
This is a standard convention in academic writing -- doesn't really need to be said